build/root/Makefile

```diff
@@ -291,7 +291,7 @@ test-cmd:
 	@echo "$$TEST_CMD_HELP_INFO"
 else
 test-cmd: generated_files
-	hack/make-rules/test-kubeadm-cmd.sh
+	source hack/lib/init.sh
```

Comment:
Why are we sourcing in the makefile instead of in the script?

Reply:
Good point, It looks sourced in `hack/make-rules/test-cmd.sh`, so this is redundant.